charmverse/app.charmverse.io
# Remember the last view for a focalboard

Steps to Test
1. Update the view of a focalboard
2. Switch to a different page
3. Navigate back to the focalboard page and see if the view is persisted

1. Move to a regular page (Needed as reloading on the same page would keep the same URL with the `viewId` query parameter)
2. Reload the page
3. Navigate to the focalboard page whose view was changed, and see if it persisted

Areas of Concern
1. Currently we are updating the local storage every time the `pages` state changes. So even if we update the icon or change the content of a page, it gets persisted to local storage which is redundant
